app/controllers/api/categories/items_controller.rb

```diff
@@ -2,10 +2,10 @@ module Api
   module Categories
     class ItemsController < ApiController
       def index
-        # chosen_item = Item.where(category: params[:category_id]) works aswell
         chosen_items = Item.all.select { |i| i.category.downcase == params[:category_id] }
```

Comment:
You should actually use the query above, with `where`. 
The difference
- `Item.all.select` fetches all Items from the database and than filters them by ID
- `Item.where(category: ...)` fetches only the right item from the database
The first option is simply wasteful and strains the database. It probably does not make a performance difference for your project, but you should learn to do it the right way.

Having said that it may not work with your production data / brake something because the option with `where` does not do apply `downcase` for the categories. To be on the safe side, look for "case-insensitive" search. It should be easy to find.

Reply:
I just realised that it applies to almost all other controllers, so change all of them (i.e. usages of `.all.select`) in another Pull Request